project/Dependencies.scala

```diff
@@ -32,7 +32,7 @@ object Dependencies {
   val aeronVersion = "1.48.7"
   // needs to be inline with the aeron version, check
   // https://github.com/aeron-io/aeron/blob/master/gradle/libs.versions.toml
-  val agronaVersion = "2.3.1"
+  val agronaVersion = "2.3.2"
```

Comment:
Please mind the comment above: agrona (apparently) needs to be in line with aeron.

This means 2.3.2 is wrong - but 2.3.1 is _also_ wrong, as https://github.com/aeron-io/aeron/blob/1.48.7/gradle/libs.versions.toml specifies 2.2.4 - i.e. we already went wrong in #2446 and #2280.

I'll exclude agrona from scala-steward (and then we just need to be careful with aeron updates...)

Reply:
Thanks for the help @raboof